src/control/lib/telemetry/telemetry.go

```diff
@@ -238,18 +238,20 @@ func (sm *statsMetric) SampleSize() uint64 {
 	return uint64(sm.stats.sample_size)
 }
 
-func startGarbageCollection(ctx context.Context) {
+func collectGarbageLoop(ctx context.Context) {
+	ticker := time.NewTicker(60 * time.Second)
```

Comment:
Oops, sorry. I should have mentioned as well that you need to `defer ticker.Stop()` in order to make sure that the ticker gets gc-ed upon leaving the loop. From https://golang.org/pkg/time/#NewTicker:

> Stop the ticker to release associated resources.

I don't think that this loop will be stopped and restarted in the lifetime of a daos_server process so I'm not too worried about it, but would be good to add in a followup just in case we change the behavior to make the loop restartable.

Reply:
Oops, my mistake. It was mentioned in the doc you linked as well, I just forgot. I can add it in a follow-on.

Reply:
I have another branch where I'm adding unit tests in this area, so I'm going to fix this in that PR.